pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -1207,6 +1208,11 @@ func doActivate(ctx *domainContext, config types.DomainConfig,
 				return
 			}
 		default:
+			persistType := vault.ReadPersistType()
+			if persistType == types.PersistZFS {
+				log.Errorf("doActivate: GetImgInfo not implemented for %s", persistType)
```

Comment:
We should be able to get information for the zvol (the size reservation etc) at this point in time.

Reply:
Sure, it is in my plan I commented on PR